backend/test_tool_calling.py

```
@@ -0,0 +1,266 @@
+    
+    # Test task creation
+    result = tool.execute(
+        title="Test Authentication Task",
+        description="Implement user authentication with JWT tokens",
+        priority="high",
+        project_id="test-project"
+    )
+    
+    print(f"Create task result: {result}")
+    
+    assert result["success"] == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `result["success"]:` for truth checks

Replace with `result["success"]`

(E712)